apps/meteor/app/apps/server/bridges/api.ts

```diff
@@ -103,7 +103,8 @@ export class AppApisBridge extends ApiBridge {
 	}
 
 	private _appApiExecutor(endpoint: IApiEndpoint, appId: string): RequestHandler {
-		return (req: IRequestWithPrivateHash, res: Response): void => {
+		// eslint-disable-next-line @typescript-eslint/no-misused-promises
+		return async (req: IRequestWithPrivateHash, res: Response): Promise<void> => {
```

Comment:
looks like there is already a `then/catch` for the `this.orch` operation, so it is not needed to convert this to `async` and disable eslint